remotepin.go

```diff
@@ -7,14 +7,21 @@ const (
 )
 
 type Pinning struct {
-	RemoteServices map[string]RemotePinningService
+	MFSRepinInterval string // in ns, us, ms, s, m, h
```

Comment:
I get why we want this to be global - it makes things a little easier for users to manage and the implementation is a bit simpler too.

WDYT about making a Policies or GlobalPolicies section here that contains both globals and potentially defaults that could be set across all implementations of a policy. As an example, if a user wants to rename the default MFS pin to be called "foo" it could be "foo" across all pinning services by default, but could be overridden by users saying "for service A call it 'fooA' and for service B call it 'barB' ".

Reply:
Having  `MFSRepinInterval` and `PinMFS` to control a single thing is not the best, will get noisy fast when we add other things.

Agree with @aschmahmann that we could simplify this. I was about to suggest that:
- `Pinning.Policies` as a struct with global defaults, keeps things tidy. It would include `MFSRepinInterval` 
- `Pinning.RemoteService[].Policies` **is the same struct**, when present overrides global

This way users don't need to track which boolean triggers what global, and can set a sensible global  + then override it for specific service.

Thoughts?

Reply:
This is generally what I'm thinking too. A couple nits to iron out though:

1) Should we scope the policies struct with substructs (e.g. MFS) or have `MFSRepinInterval`?
2) Would the fact that a particular policy (e.g. MFS pinning) is enabled/disabled be in the global?

Reply:
(we decided to not have globals and use `RemotePinningServiceMFSPolicy` struct per each service)